docs/data/reads.md

```diff
@@ -0,0 +1,39 @@
+---
+id: reads
+title: Reads
+sidebar_label: Reads
+---
+
+ICGC ARGO provides access to data from ARGO member programs generated through the standardized [ARGO Analysis pipelines](/docs/analysis_workflows/data-types). The following page lists genomic read analyses and data files generated by the ARGO Analysis pipelines.
+
+## Submitted Reads
+
+Raw reads submitted by ARGO member programs are used as inputs within the ARGO analytic pipeline. Data is accepted in FASTQ or BAM (aligned or unaligned) formats, must be compressed, and can be aligned or unaligned.
+
+### File Types
+
+| Filename Pattern        | Description                             | Analysis Type    | Data Category    | Generating Workflow(s) |
+| ----------------------- | --------------------------------------- | ---------------- | ---------------- | ---------------------- |
+| \*.fq.gz, \*.fastq.gz   | FASTQ file format compressed with gzip  | sequencing_reads | Sequencing Reads | N/A                    |
+| \*.fq.bz2, \*.fastq.bz2 | FASTQ file format compressed with bzip2 | sequencing_reads | Sequencing Reads | N/A                    |
+| \*.bam                  | Binary Alignment Map (BAM) file format  | sequencing_reads | Sequencing Reads | N/A                    |
+
+## Aligned Reads
+
+A read is a sequence obtained from a single sequencing experiment. An aligned read is a sequence that has been aligned to a common reference genome. Typically these reads can number from the hundreds of thousands to tens of millions. Aligned reads are generated internally by the ARGO alignment workflow(s) and maintained in CRAM (a compressed version of BAM that stores only reads different from the reference sequence) format. ARGO reads are aligned to the GRCh38 Human Reference Genome.
```

Comment:
CRAM is defined here...is this the first instance of it? Like should it be defined when used on the other page too?

Reply:
yes, originally this was all on one page. this is a good question... hmm i lean towards leaving the definition here as the other pages can reference this?

Reply:
sure..or both pages if you want

Reply:
hmm i think that its best just to leave it - the qc files are all past the aligned stage, so i think people already know what a CRAM is at this point.  lets add later if needed.